WPMedia/fusion-news-theme-blocks
# [Themes 1676] Fixed triple-chain large-manual-promo Desktop view

## Description

#### Jira Ticket: [THEMES-1676](https://arcpublishing.atlassian.net/browse/THEMES-1676)


## Test Steps

1. Checkout this branch `git checkout THEMES-1676`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/triple-chain-block`
3. Add large-manual-promo-blocks to triple-chain. 
4. Check for Desktop view, Large-manual-promo Image and description are side-by-side in desktop view


[THEMES-1676]: https://arcpublishing.atlassian.net/browse/THEMES-1676?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
